src/misc.test-d.ts

```diff
@@ -101,14 +134,14 @@ expectNotAssignable<RuntimeObject>(Symbol('test'));
 // The RuntimeObject type gets confused by interfaces. This interface is a valid object,
 // but it's incompatible with the RuntimeObject type.
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-interface A {
+interface RuntimeObjectInterfaceExample {
```

Comment:
This was renamed to avoid a variable name conflict with the new test cases.

These names seem better anyway.